src/main/java/de/hpi/bpt/argos/common/NullRestRequest.java

```diff
@@ -0,0 +1,76 @@
+package de.hpi.bpt.argos.common;
+
+import de.hpi.bpt.argos.api.response.ResponseFactory;
+
+/**
+ * {@inheritDoc}
+ * This is a null implementation.
+ */
+public class NullRestRequest implements RestRequest {
+
+	protected String content;
+	protected String response;
+
+	/**
+	 * This constructor initializes the members with default values.
+	 */
+	public NullRestRequest() {
+		response = "null request";
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public void setMethod(String method) {
+		// empty, since it has no effect
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public void setProperty(String key, String value) {
+		// empty, since it has no effect
```

Comment:
I found it on this great page: http://userpage.fu-berlin.de/~ram/pub/pub_jf47ht81Ht/java_leere_anweisung_de !